src/ORM/Query.php

```diff
@@ -394,6 +393,29 @@ public function contain($associations = null, $override = false)
     }
 
     /**
+     * Clears the contained associations from the current query.
+     *
+     * @return $this
+     */
+    public function clearContain()
+    {
+        $this->getEagerLoader()->clearContain();
+        $this->_dirty();
+
+        return $this;
+    }
+
+    /**
+     * Gets the contained associations
+     *
+     * @return array
+     */
+    public function getContain()
```

Comment:
I'm not putting that effort in until somebody decides to keep it or to remove it.